service/history/outbound_queue_standby_task_executor.go

```diff
@@ -157,6 +157,15 @@ func (e *outboundQueueStandbyTaskExecutor) processTask(
 
 	historyResendInfo, err := actionFn(ctx)
 	if err != nil {
+		if errors.Is(err, consts.ErrTaskRetry) {
+			// If the error is retryable, it means the task hasn't been processed on the active side yet.
+			// Since the most likely reason would be the destination is down, we wrap the error here with
+			// DestinationDownError so it can trigger the circuit breaker on the standby side as well.
+			// Assuming the dynamic config StandbyTaskMissingEventsDiscardDelay is long enough, it should
+			// give enough time for the active side to execute the task successfully, and the standby side
+			// to process it as well without discarding the task.
+			err = queues.NewDestinationDownError("standby task executor returned retryable error", err)
```

Comment:
Can we add a dc flag for this? We can default to this behavior but in case things went wrong, we may still want to force drop them as a mitigation.

Reply:
Added a dynamic config the task type filter. Let me know if it should be a different filter.

Reply:
+1 to Roey's comment above. Namespace + Destination filters makes more sense to me. 
I am not exactly sure why those delay configs are task type based. Maybe @yux0 have some context?